flake.nix

```diff
@@ -0,0 +1,125 @@
+{
```

Comment:
> [!NOTE]
Should note that I had to remove a few checks like `cargoAudit`. This was before adding binary to packages array, so that might have been the culprit may actually work now as of writing this.